src/tests/ftest/interoperability/diff_versions.yaml

```diff
@@ -23,7 +22,6 @@ container:
 attrtests:
   num_attributes: 20
 dfuse:
-  mount_dir: "/tmp/daos_dfuse1/"
```

Comment:
This will cause problems with the test as it does a `mount_dir = self.params.get("mount_dir", '/run/dfuse/*')`